File: SPOJ/GLJIVE.cpp
#include <iostream>
#include <stdio.h>
using namespace std;

int main(){
	int sum,min,input,count;
	sum = min = count = 0;
	while (count < 10) {
		scanf("%d", &input);
		sum += input;
		if (sum == 100){
			cout << 100 <<endl;
			break;
		}
		else if(sum > 100){
			if ((sum - 100) == (100 - min)){
				cout <<sum <<endl;
			}
			else if((sum - 100) < (100 - min)){
				cout <<sum <<endl;
			}
			else{
				cout << min << endl;
			}
			break;
		}	
		else{
			min = sum;
		}
		count++;
	}
	if (sum < 100){
		cout << sum << endl;
	}
	return 0;
}

#DONE

File: SPOJ/GLJIVE.py
sum = 0
min = 0
while True:
	try:
		string = raw_input()
		if (string == "EOF"):
			break
		sum += int(string)
		if sum == 100:
			print 100
			break
		if (sum > 100):
			if ( sum - 100) == (100 - min):
				print sum
			elif ( sum - 100) > (100 - min) :
				print min
			else:
				print sum
			break
		min = sum
		raw_input()
	except:
		break